OrbitClientServices/include/OrbitClientServices/ProcessManager.h

```diff
@@ -5,6 +5,8 @@
 #ifndef ORBIT_CLIENT_SERVICES_PROCESS_MANAGER_H_
 #define ORBIT_CLIENT_SERVICES_PROCESS_MANAGER_H_
 
+#include <tracepoint.pb.h>
```

Comment:
This should be placed after
#include "symbol.pb.h"
and it should use quotes("") insted of brakets(<>)